components/policies/PolicyHistory.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { usePaginatedFetch } from "hooks/usePaginatedFetch";
+import Loading from "components/Loading";
+import LabelWithValue from "components/LabelWithValue";
+import dayjs from "dayjs";
+import { DATE_TIME_FORMAT } from "utils/constants";
+import Button from "components/Button";
+import styles from "styles/modules/PolicyHistory.module.scss";
+import { useTheme } from "providers/theme";
+
+const PolicyHistory = ({ policy }) => {
+  const { theme } = useTheme();
+  const { data, loading, isLastPage, goToNextPage } = usePaginatedFetch(
+    `/api/policies/${policy.id}/versions`,
+    {},
+    15
+  );
+
+  return (
+    <div className={styles[theme]}>
+      <Loading loading={loading}>
+        {data && (
+          <>
+            {data.map((version, index) => {
+              return (
+                <div key={version.version} className={styles.versionCard}>
+                  <div>
+                    <LabelWithValue
+                      label={"Policy Version"}
+                      value={`${version.version}${
```

Comment:
Could be nice to display `v1`, `v2`, etc here but I don't feel strongly about it.

Reply:
![image](https://user-images.githubusercontent.com/25139211/122818221-8b612f80-d29e-11eb-98f2-d2cab90f892c.png)
Does it look weird to have `Policy Version` still? I'm worried just `v1` won't be enough to connect the dots until you have multiple versions

Reply:
I'm gonna go with showing both the label `Policy Version` and `v#`

Reply:
I don't think it's too weird, but I see why it might be redundant. I'm fine either way